pkg/testing/log.go

```diff
@@ -0,0 +1,23 @@
+package testing
```

Comment:
You don't need this file, see my previous comment explaining how to use [this function](https://github.com/asticode/go-astikit/blob/master/logger.go#L183)

Reply:
https://github.com/asticode/go-astilectron/pull/358#discussion_r835037631